tests/test_formatter/__snapshots__/test_formatter_output.ambr

```diff
@@ -957,9 +957,10 @@
   '''
   
   [4m[1mtests/fixtures/notebook.ipynb[0m[0m
+    0:1      WPS453 Found executable mismatch: file is executable but no shebang is present
```

Comment:
this does not look related.

Reply:
have no idea how it happens. i`m just run 
poetry run pytest --snapshot-update
and all test is passed

Reply:
is it bad idea to fix this manually?

Reply:
it is fine, just remove the line

Reply:
i think this happens because of wsl. but have no idea why